sycl/test-e2e/Basic/fpga_tests/fpga_queue.cpp

```diff
@@ -31,7 +31,7 @@ void GetCLQueue(event sycl_event, std::set<cl_command_queue> &cl_queues) {
     assert(CL_SUCCESS == error && "Failed to obtain queue from OpenCL event");
 
     cl_queues.insert(cl_queue);
-  } catch (invalid_object_error e) {
+  } catch (const exception &e) {
```

Comment:
Should we also check error code here now?

Reply:
It doesn't look like this exceptional path is supposed to be happening during successful execution of the test (we probably would fail later checks if so). As such, I don't think we need to check the error code here - that piece of code seems to be used for the debug prints during failed execution.